src/app/components/App.jsx

```diff
@@ -465,12 +476,18 @@ export default connect(
     },
     dispatch => ({
         loginUser: () => dispatch(userActions.usernamePasswordLogin({})),
-        depositSteem: username => {
+        openBlocktrades: username => {
             const new_window = window.open();
             new_window.opener = null;
             new_window.location =
                 'https://blocktrades.us/?input_coin_type=eth&output_coin_type=steem&receive_address=' +
```

Comment:
A little off piste - One thing I would add here is a check that the username is set, otherwise this passes `undefined`  as a query string.